ocpmodels/datasets/trajectory_lmdb.py

```diff
@@ -29,13 +29,15 @@ def __init__(self, config):
         ]
 
         self.inds = []
+        self.transform = None
 
     def __len__(self):
         return len(self._keys)
 
     def __getitem__(self, idx):
         datapoint_pickled = self.db_txn.get(self._keys[idx])
         data_object = pickle.loads(datapoint_pickled)
+        data_object = data_object if self.transform is None else self.transform(data_object)
```

Comment:
(Make sure to `pre-commit install` and run this through all the pre-commit hooks.)

Reply:
Fixed!